src/components/MapView/Analyser/index.tsx

```diff
@@ -98,12 +100,15 @@ function Analyser({ classes }: AnalyserProps) {
           (LayerDefinitions[hazardLayerId] as WMSLayerProps)?.serverLayerName
         ]
       : null;
+    if (Number(new URLSearchParams(history.location.search).get('date'))) {
```

Comment:
I don't think this is a good idea. The goal of this useEffect is to load default dates when a new hazard layer is selected. We will now disable this feature if there's something in the url.

Reply:
The goal of this if statement is to check if the `URL`  has a date in it, ignore the code within the `useEffect` which actually mess up with the URL dates and you don't get the expected result.

Reply:
We still need to find a better way to do this. 

If I run an analysis, and copy paste the url to a new page, then try editing the hazard layer field, the dates won't update like they're supposed to. The approach I mentioned below (https://github.com/WFP-VAM/prism-frontend/pull/120#issuecomment-753710241) could work here. Otherwise, if there's a solution you can think of which still keeps this feature working, feel free to include it :+1:

Reply:
One I can think of off the top of my head is using a new boolean state to dictate if the state was set by the user or by the computer. If it was set by the computer, we can ignore this useEffect, otherwise we should still run this code. Hopefully this is helpful enough as a jumping point